hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/TableMapReduceUtil.java

```diff
@@ -819,7 +819,6 @@ public static void addHBaseDependencyJars(Configuration conf) throws IOException
       org.apache.hbase.thirdparty.com.google.protobuf.UnsafeByteOperations.class, // hb-sh-protobuf
       org.apache.hbase.thirdparty.io.netty.channel.Channel.class,    // hbase-shaded-netty
       org.apache.zookeeper.ZooKeeper.class,                          // zookeeper
-      com.google.protobuf.Message.class,                             // protobuf
```

Comment:
Why this removed? This affect what?

Reply:
You can see the comment on jira. The maven shade plugin will also relocate this line and then cause class not found error. I tried to not relocated protobuf, but then it leads to other problems when building hbase-shaded-client. So I suppose there are still other problems. But anyway, for master, we do not depend on protobuf any more, in hbase we all use our thirdparty protobuf, so here we could just remove it. And removing it can fix the class not found problem.

Reply:
https://issues.apache.org/jira/browse/HBASE-25049